website/docs/examples/motor_state/README.md

```diff
@@ -5,8 +5,20 @@ title: Motor State
 
 # Amiga Motor State Stream Example
 
-Currently this is a very basic [example](https://github.com/farm-ng/farm-ng-amiga/blob/main-v2/py/examples/motor_states_stream/main.py)
-showing how to access and decode the
+:::info Basic Knowledge Requirements
+
+Before diving into this code, here's a quick heads-up on what you'll need to be familiar with:
+
+1. **Python Programming**: It's important to have a good grasp of Python, especially with concepts
```

Comment:
i think this you should be a general requirements of the whole sdk and we assume everyone landing here know  python basics

Reply:
Sounds fair to me. But then, what's the best way of doing it? Should we add an admonition in the [Examples Index](https://amiga.farm-ng.com/docs/examples/examples-index) and then remove this "Python" requirement from all examples? @edgarriba (cc @Hackerman342)